### DISCUSSION TITLE
_bluetooth.error: (2, 'No such file or directory')

### DISCUSSION THREAD

**Discussion by Em-7dice** [rootkiwi/an2linuxserver#71] [Q&A]
So I'm looking to run this on a raspberry pi 0w via bluetooth, and I'm getting this error on a completely fresh install of the latest raspberry pi OS. I've checked all the dependancies, and I have them all. Idk whats up. Attatched is the error message I keep getting and all its details. It seems that the python script is trying to point to a serial profile or smth that doesn't exist. Any idea what's up with that or how to fix it?

**Comment by d7415** [comment]
It looks like the error message didn't attach.

You may want to look at [This README](https://github.com/rootkiwi/an2linuxserver/blob/58648ecbdc9d62d3e3e28484e3e7a1f12507dfdf/init/systemd-system/README.md), #57 and #31.
